src/content/docs/journey-app-slow/create-benchmarks.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: 'Create performance benchmarks with service levels'
+metaDescription: 'Learn how to create and define performance benchmarks with service levels.'
+---
+
+import sonicSliUiOverview from 'images/sonic_screenshot-full_sli-ui-overview.png'
+
+## Objectives [#objectives]
+
+By the end of this tutorial, you will:
+
+* Understand service level indicators (SLIs) and service level obectives (SLOs).
+* Have created and defined SLIs and SLOs for your front-end experience.
+* Have alerts set up to warn of any SLI or SLO breaches so you can react before they impact your users.
+
+## Why create performance benchmarks [#why]
+
+Now that you've remdiated the root cause of your app's slowness, let's set up service levels so you can respond quickly if any slowness appears again. 
+
+Let's look at the difference between service levels, SLOs and SLIs. Service levels describe services provided to users within a given period of time, in measurable terms. Service level objectives (SLOs) are the goals set for the availability expected out of a system. Service level indicators (SLIs) are the key measurements and metrics to determine the availability of a system. 
+
+Defining and managing your service levels following the steps below provides the ability to: 
+
+* Ease future setup: Automatically establish a baseline of performance and reliability for any service with a one-click setup.
+* Define reliability across teams: Avoid arduous alignment processes with SLO and SLI recommendations that help you determine service boundaries. Set reliability benchmarks automatically based on recent performance metrics in any entity.
+* Iterate and improve: With full-stack context and automation through open-source infrastructure-as-code tools like Terraform, teams have insight into how specific nodes or services impact system reliability and can quickly take control over their performance. Custom views for both service owners and business leaders drive operational efficiency and lead to better reporting, alerting, and incident management processes.
+* Standardize reliability: Cross-organizational teams have a unified, transparent view of service reliability, and can better comply with customer-facing SLAs. SLO compliance metrics and error budgets give organizations a way to report on reliability and implement changes across applications, infrastructure, and teams in a cohesive fashion.
+
+## Create performance benchmarks [#create]
+
+Let's get started with creating your front-end performance benchmarks:
+
+<Steps>
+    <Step>
+
+    ## Select service level indicators:
```

Comment:
A few issues here:

* Why is this the only step with a header?
* Why is it an h2 when step-headers in other tutorials are h3s?
* Why is there a colon?